department-of-veterans-affairs/va.gov-cms
# VACMS-4845: Update content creator perms.

## Description
See #4845

## Testing done
Visual / phpunit

## QA steps
- As user 1414, go to /node/add/
- Visually verify you do not see VAMC Facility and Publication among the options

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
